tests/dummy/app/templates/examples.hbs

```diff
@@ -1,90 +1,113 @@
-
-
 Simple use case:
 {{code-snippet name='simple-use.hbs'}}
 
 <div class='container'>
   {{imgix-image
-    path='/examples/puffins.jpg'
-    class='header-img'
-    fit='crop'
+  path='/examples/puffins.jpg'
```

Comment:
Is this indentation change intentional? Not a big deal, but I think it's less readable this way.

Reply:
I just let Prettier do it's thing. So unintentionally intentional? 🤔

Reply:
So turns out it wasn't Prettier, but the default VS Code formatter. I've disabled this and I formatted the file manually.